### PR TITLE
Order api get

### PR DESCRIPTION
**Pull request by RachelLivne**
order-api-get/create/edut

### CODE REVIEW

File: src/main/java/com/yellow/ordermanageryellow/Service/OrdersService.java [hunk 1]
```diff
@@ -1,23 +1,71 @@
...
+
+        Pageable pageable = PageRequest.of(pageNumber, pageSize/* pageSize parameter omitted */, sort);
+
+        Page<Orders> pageOrders = OrdersRepository.findByCompanyId_IdAndOrderStatusIdAndEmployee(companyId, status, userId, pageable);
+        return pageOrders.getContent();
+
     }
+
+    public String insert(Orders newOrder) {
+        if (newOrder.getOrderStatusId() != "new" || newOrder.getOrderStatusId() != "approve") {
```
Comment (moriyahazan) [inline src/main/java/com/yellow/ordermanageryellow/Service/OrdersService.java:46]: Status should be enum

File: src/main/java/com/yellow/ordermanageryellow/controller/OrderController.java [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import com.yellow.ordermanageryellow.model.Orders;
+
+
+import java.util.List;
+import java.util.NoSuchElementException;
+
+@RestController
+@RequestMapping("/Order")
```
Comment (moriyahazan) [inline src/main/java/com/yellow/ordermanageryellow/controller/OrderController.java:16]: please change to /order

File: src/main/java/com/yellow/ordermanageryellow/Service/OrdersService.java [hunk 3]
```diff
@@ -1,23 +1,71 @@
...
+import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.Optional;
 
 
 @Service
-public class OrdersService implements CommandLineRunner {
+public class OrdersService {
+    @Autowired
     private final OrdersRepository OrdersRepository;
```
Comment (moriyahazan) [inline src/main/java/com/yellow/ordermanageryellow/Service/OrdersService.java:22]: change to start wit lower letter

**Review by moriyahazan — APPROVED** [review]
very clean code!
some small comments please fix

**Comment by moriyahazan** [comment]
Please update from merge, solve conflict carefully and merge to main

**Comment by RachelLivne** [comment]
order-api

**Pull request merged**